utils/m0reportbug

```diff
@@ -159,7 +159,7 @@ main() {
     # Helps in debugging, If m0reportbug fails due to huge size
     echo "du -sh $LIVE_CORE_DIR" >&2
```

Comment:
Also check here if LIVE_CORE_DIR exists or not.

Reply:
@swapnil-seagate  Do we collect live cores on k8s setup? I think to collects livecore we need gcore (gdb package) which is not installed by default inside containers now.  So I doubt about it.

Reply:
I dont know if we collect live cores or not on k8s setup, but this check is added for validating if the directory exists or not.